src/index.js

```diff
@@ -237,7 +247,7 @@ VirtualMachine.prototype.emitTargetsUpdate = function () {
             return [target.id, target.getName()];
         }),
         // Currently editing target id.
-        editingTarget: this.editingTarget.id
+        editingTarget: this.editingTarget ? this.editingTarget.id : null
```

Comment:
Due to the above, `editingTarget` is sometimes `null` when `emitTargetsUpdate` is called.